### PR TITLE
Test function sqlite

### PR DESCRIPTION
**Pull request by ashishpriyadarshiCIC**

### CODE REVIEW

**Comment by ashishpriyadarshiCIC** [comment]
@apoorvaeternity @henrykironde Ready for review

**Comment by henrykironde** [comment]
Looks good and ready for a good review. I think we shall talk about the names of the functions. sqlite_to_csv and sqlite2csv

```python
def sqlite_to_csv(self, src_path, path_to_csv, table_name=None, encoding=ENCODING):
         sqlite2csv(
```

File: retriever/lib/engine_tools.py [hunk 1]
```diff
@@ -132,6 +135,20 @@ def json2csv(input_file, output_file=None, header_values=None, encoding=ENCODING
     return output_file
 
 
+def sqlite2csv(input_file, output_file, table_name=None, encoding=ENCODING):
+    """Convert sqlite database file to CSV.
+
+    Function is used for only testing and can handle the file of the size.
+    """
+    conn = sql.connect(input_file)
+    cursorObj = conn.cursor()
```
Comment (apoorvaeternity) [inline retriever/lib/engine_tools.py:144]: snake case for variable name

**Review by apoorvaeternity — APPROVED** [review]
LGTM

File: scripts/portal_project_teaching.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
+{
+    "citation": "",
+    "description": "The Portal Project Teaching Database is a simplified version of the Portal Project Database designed for teaching",
+    "encoding": "utf-8",
+    "homepage": "https://figshare.com/articles/Portal_Project_Teaching_Database/1314459",
+    "keywords": ["mammals", "desert", "time-series", "experimental", "observational"],
+    "licenses": [],
+    "name": "portal-project-test",
```
Comment (henrykironde) [inline scripts/portal_project_teaching.json:8]: The name and the file name differ `portal-project-teaching `

**Comment by henrykironde** [comment]
Thanks @ashishpriyadarshiCIC 👍

**Pull request merged**